cartaya/4/style4.css

```diff
@@ -0,0 +1,213 @@
+@font-face {
+    font-family: 'HelveticaNeueCE35Thin';
+    src: url('Helvetica Neue CE 35 Thin.ttf');
+}
+
+* {
+    margin: 0px;
+}
+
+body {
+    overflow: auto;
+}
+
+.section-left, .section-right {
+    border: 1px solid black;
+    border-color: #838282; 
+    background-color: #FAFAFA;
+    float: left;
+    margin-top: 50px;
+}
+
+.section-left {
+    width: 613px;
+    height: 496px;
+    padding-left: 35px;
+    padding-top: 70px;
+    padding-bottom: 29px;
+    position: relative;
+    margin-left: 35px;
+}
+
+.section-right {
+    width: 650px;
+    height: 563px;
+    padding-left: 20px;
+    padding-top: 32px;
+    margin-top: 49.5px;
+    background-color: white;
+    position: relative;
+    left: -40px;
+    z-index: 1;
+}
+
+.section-left p, .section-left address  {
+    display: inline-block;
+}
+
+.title {
+    font-family: HelveticaNeueCE35Thin;

```

Comment:
You can define the font-family property  in the element container, this is a Inherited property you don’t need specify in each element inside container. If all the texts in the document have the same font, define this property only in the body or element container.